src/metadata/metadata.c

```diff
@@ -1204,10 +1204,13 @@ static void _recovery_reset_cline_metadata(struct ocf_cache *cache,
 	ocf_metadata_set_core_info(cache, cline, OCF_CORE_MAX, ULLONG_MAX);
 
 	metadata_clear_valid(cache, cline);
+	metadata_clear_dirty(cache, cline);
```

Comment:
You can use `metadata_init_status_bits()` instead

Reply:
it's done, thanks